src/normalizer/mod.rs

```diff
@@ -36,6 +36,7 @@ pub struct NormalizedTokenIter<'o> {
     token_iter: Box<dyn Iterator<Item = Token<'o>> + 'o>,
     inner: Box<dyn Iterator<Item = Token<'o>> + 'o>,
     normalizer: &'static Box<dyn Normalizer>,
+    options: Option<NormalizerOption>,
```

Comment:
I think `Option` enum is not necessary when `NormalizerOption` implements `Default`